src/panels/config/system-health/ha-config-system-health.ts

```diff
@@ -37,15 +35,38 @@ const sortKeys = (a: string, b: string) => {
   return 0;
 };
 
-class SystemHealthCard extends LitElement {
+@customElement("ha-config-system-health")
+class HaConfigSystemHealth extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
+  @property({ type: Boolean }) public narrow!: boolean;
+
   @state() private _info?: SystemHealthInfo;
 
-  protected render(): TemplateResult {
-    if (!this.hass) {
-      return html``;
+  protected firstUpdated(changedProps) {
+    super.firstUpdated(changedProps);
+
+    this.hass!.loadBackendTranslation("system_health");
+
+    if (!isComponentLoaded(this.hass!, "system_health")) {
```

Comment:
Is `system_health` is all we are showing, we should do the filtering in the menu, by setting `component: "system_health"`